### ISSUE TITLE
Homogenize docstrings of class methods

### ISSUE THREAD

**Issue by alebrigant** [geomstats/geomstats#1631]
### Where should the content be modified?

In the docstrings of class methods.

### What should be modified?

Some docstrings titles are the name of the object to compute, e.g.
```
def metric_matrix(self, base_point=None):
        """Metric matrix at the tangent space at a base point.
```
while others start with "compute", e.g.
```
 def inner_product_derivative_matrix(self, base_point=None):
        """Compute derivative of the inner prod matrix at base point.
```

**Comment by ninamiolane** [comment]
I agree that we should uniformize! However, we should keep the "compute" as per pep8 guidelines: docstrings' one-liners should start with verbs in infinitive form. Flake8 will complain if not :)

**Comment by alebrigant** [comment]
Ok I was not aware of this guideline ! Then I guess we should instead add "compute" of other verbs in infinite form where they are missing.

**Comment by lpereira95** [comment]
@nanjekyejoannah, I think we can address this soon.
